packages/updating-element/src/updating-element.ts

```diff
@@ -27,6 +27,13 @@ import {
 } from './css-tag.js';
 export * from './css-tag.js';
 
+const DEV_MODE = true;
+if (DEV_MODE) {
+  console.warn(
+    `updating-element is in dev mode. Not recommended for production!`
```

Comment:
Maybe strong wording than "Not recommended"? Like "Do not use in production!"

It'd also be great to include a link to docs on how to fix this. TODO?

Reply:
Done.